src/puppetlabs/trapperkeeper/services.clj

```diff
@@ -22,6 +22,8 @@
   (service-context [this] "Returns the context map for this service")
   (get-service [this service-id] "Returns the service with the given service id")
   (get-services [this] "Returns a sequence containing all of the services in the app")
+  (call-service-fn [this other-service-id fn-name args default] "Calls named function in other service, returning default if that service is not included.")
```

Comment:
Perhaps it'd be nice to switch the order of `args` and `default`, which should remove the requirement that callers wrap `args` in a list ... assuming that protocols support `& args` syntax, which I am not sure if they do.